docs/how_to/use_sampler.rst

```diff
@@ -0,0 +1,248 @@
+
+The :class:`~qiskit.primitives.Sampler` primitive can be run with unbound parameterized circuits like the one below.
+You can also manually bind values to the parameters of the circuit and follow the steps
+of the previous example.
+
+.. testcode::
+
+    from qiskit.circuit import Parameter
+
+    theta = Parameter('θ')
+    param_qc = QuantumCircuit(2)
+    param_qc.ry(theta, 0)
+    param_qc.cx(0,1)
+    param_qc.measure_all()
+    print(param_qc.draw())
+
+.. testoutput::
+
+            ┌───────┐      ░ ┌─┐   
+       q_0: ┤ Ry(θ) ├──■───░─┤M├───
+            └───────┘┌─┴─┐ ░ └╥┘┌─┐
+       q_1: ─────────┤ X ├─░──╫─┤M├
+                     └───┘ ░  ║ └╥┘
+    meas: 2/══════════════════╩══╩═
+                              0  1 
+
+The main difference from the previous case is that now you need to specify the sets of parameter values
+for which you want to evaluate the expectation value as a ``list`` of ``list``\ s of ``float``\ s.
+The idea is that the ``i``-th element of the outer ``list`` is the set of parameter values
+that corresponds to the ``i``-th circuit.
+
+.. testcode::
+
+    import numpy as np
+
+    parameter_values = [[0], [np.pi/6], [np.pi/2]]
+
+    job = sampler.run([param_qc]*3, parameter_values=parameter_values)
+    dists = job.result().quasi_dists
+
+    for i in range(3):
+        print(f"Parameter: {parameter_values[i][0]:.5f}\t Probabilities: {dists[i]}")
+
+.. testoutput::
+
+    Parameter: 0.00000	 Probabilities: {0: 1.0}
+    Parameter: 0.52360	 Probabilities: {0: 0.9330127018922194, 3: 0.0669872981077807}
+    Parameter: 1.57080	 Probabilities: {0: 0.5000000000000001, 3: 0.4999999999999999}
+
+Change run options
+==================
+
+Your workflow might require tuning primitive run options, such as shots.
+
+By default, the reference :class:`~qiskit.primitives.Sampler` class performs an exact statevector
+calculation based on the :class:`~qiskit.quantum_info.Statevector` class. However, this can be 
+modified to include shot noise if the number of ``shots`` is set. 
+For reproducibility purposes, a ``seed`` will also be set in the following examples.
+
+There are two main ways of setting options in the :class:`~qiskit.primitives.Sampler`:
+
+* Set keyword arguments in the :meth:`~qiskit.primitives.Sampler.run` method.
+* Modify :class:`~qiskit.primitives.Sampler` options.
+
+Set keyword arguments for :meth:`~qiskit.primitives.Sampler.run`
+----------------------------------------------------------------
+
+If you only want to change the settings for a specific run, it can be more convenient to
+set the options inside the :meth:`~qiskit.primitives.Sampler.run` method. You can do this by
+passing them as keyword arguments.
+
+.. testcode::
+
+    job = sampler.run(qc, shots=2048, seed=123)
+    result = job.result()
+    print(result)
+
+.. testoutput::
+
+    SamplerResult(quasi_dists=[{0: 0.5205078125, 3: 0.4794921875}], metadata=[{'shots': 2048}])
+
+Change :class:`~qiskit.primitives.Sampler` options
```

Comment:
Similar to https://github.com/Qiskit/qiskit-terra/pull/9716#discussion_r1189704507

Reply:
Done in [254c8de](https://github.com/Qiskit/qiskit-terra/pull/9716/commits/254c8de822ab329082b7efd78b57377c74d2c495)